src/server.c

```diff
@@ -1515,6 +1536,9 @@ int serverCron(struct aeEventLoop *eventLoop, long long id, void *clientData) {
                           &ei);
 
     server.cronloops++;
+
+    server.el_cron_duration = getMonotonicUs() - server.el_cron_duration;
```

Comment:
If another variable was used above, this could be written like
```
server.el_cron_duration_us = elapsedUs(el_cron_start);
```
Note the rename of `el_cron_duration` to `el_cron_duration_us` - it's helpful to tag units to time duration variables.